subsys/bluetooth/host/Kconfig

```diff
@@ -179,6 +179,8 @@ config BT_HOST_CRYPTO
 config BT_HOST_CRYPTO_PRNG
 	bool "Use PSA crypto API library for random number generation"
 	default y
+	# An implementation of z_impl_sys_rand_get() is required
+	select ENTROPY_GENERATOR if !TEST_RANDOM_GENERATOR
 	select PSA_WANT_ALG_SHA_256
```

Comment:
I know this is a comment on a pre-existing code (that I did btw), but since you are at this I think we can drop `PSA_WANT` below because HMAC-DRBG was only used in TinyCrypt to generate random data, while with PSA Crypto API we call `psa_generate_random()` directly.
If you think that this does not belong to this PR, then I can open a separate one ;)

Reply:
Which one of the `PSA_WANT`? All of them?

Reply:
Yes, all of them. The rationale is:
- if `psa_generate_random()` can rely on some entropy generator, then no crypto algorithm is required on top of that;
- if a valid entropy source is not available, `psa_generate_random()` will fall back to legacy mode (`MBEDTLS_PSA_CRYPTO_LEGACY_RNG`) where all the required legacy should be automatically selected.